### DISCUSSION TITLE
Create likert scale using picture

### DISCUSSION THREAD

**Discussion by Nega100** [jspsych/jsPsych#1216] [Q&A] [answered]
Hello Josh and all Contributors,
I work with children for several years, for some reason I start to use this incredible library jspsych. However when I tried to using two plugins together (or create one) that came out with error. What I am trying to do is create pictorial personality trait questionnaire for children. So there will be picture in each anchor likert scale. It is possible Josh to create such scale.

Thanks
Nega

**Comment by jodeleeuw** [comment, marked as answer]
Hi Nega,

I think the simplest way to do this with the current set of plugins is to use the `html-slider-response` plugin and make your stimulus two images that are aligned to the left and right of the page. Something like this:

```js
var trait_question = {
	type: 'html-slider-response',
	stimulus: `<div style="width:800px; height: 200px; position: relative;">
				<img src="leftSideImage.png" style="height:200px; left:0px; position: absolute;"></img>
				<img src="rightSideImage.png" style="height:200px; right:0px; position: absolute;"></img>
			</div>`,
	slider_width: 800
}
```

You could use the [other parameters](https://www.jspsych.org/plugins/jspsych-html-slider-response/) of the slider-response plugin to control things like how many unique steps there are in the scale if you want to make it more like a Likert scale and less like a continuous rating.

**Reply by Nega100** [reply]
Yes thank you very much Josh!! This is what I needed
